### PR TITLE
infer placeholder param types

### PR DESCRIPTION
**Pull request by gottfired**

### CODE REVIEW

File: template.json [hunk 1]
```diff
@@ -38,6 +38,7 @@
       "@types/react-router-dom": "^5.3.1",
       "@types/webpack-env": "1.18.0",
       "@types/jest": "^29.5.1",
+      "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
```
Comment (chrztoph) [inline template.json:41]: Why is this needed?

Comment (gottfired) [reply template.json:41]: I ran into this. Basically CRA is dead https://github.com/facebook/create-react-app/issues/13216

Comment (chrztoph) [reply template.json:41]: Hmm someone fixed it here: https://github.com/facebook/create-react-app/commit/0f5e990b8a04f53861d64ff53751517bbf73d867

But looks like no new versions have published in a while.

File: template/src/i18n/util.tsx [hunk 2]
```diff
@@ -63,12 +108,18 @@ export const tHtml = (messageId: MessageIDS, values?: Record<string, any>) => {
...
             values={{
-                b: (chunks) => {
-                    return <b>{chunks}</b>;
-                },
-                br: () => {
-                    return <br />;
-                },
+                b: (chunks) => (
+                    <b>
+                        <>{chunks}</>
```
Comment (schneckentempo) [inline template/src/i18n/util.tsx:113]: why do we suddenly need fragments here?

Comment (schneckentempo) [reply template/src/i18n/util.tsx:113]: lgtm then
